etc/behat/services/contexts/ui.xml

```diff
@@ -38,6 +39,15 @@
             <tag name="sylius.behat.context" />
         </service>
 
+        <service id="sylius.behat.context.ui.admin.managing_currency" class="%sylius.behat.context.ui.admin.managing_currency.class%" scope="scenario">
+            <argument type="service" id="sylius.behat.page.admin.currency.index" />
+            <argument type="service" id="sylius.behat.page.admin.currency.create" />
+            <argument type="service" id="sylius.behat.page.admin.currency.update" />
+            <argument type="service" id="sylius.behat.notification_checker" />
+            <argument type="service" id="sylius.behat.current_page_resolver" />
```

Comment:
I would suggest to put `notification_checker` after `current_page_resolver`. At least we have such an order under ManagingTaxRatesContext